src/formats/game_model.cpp

```diff
@@ -91,16 +91,32 @@ void moby_model::read() {
 		submodel.vertices.resize(vertex_header.vertex_count_2 + vertex_header.vertex_count_4 + vertex_header.main_vertex_count);
 		_backing.seek(entry.vertex_offset + vertex_header.vertex_table_offset);
 		_backing.read_v(submodel.vertices);
-		
+
 		// This is almost certainly wrong, but makes the models look better for the time being.
 		if(submodel.vertices.size() > 0) {
 			for(std::size_t i = 0; i < vertex_header.vertex_count_8; i++) {
 				submodel.vertices.push_back(submodel.vertices.back());
 			}
 		}
 		
-		if(!validate_indices(submodel)) {
```

Comment:
I removed this function and moved the contents to here such that looping through the indexes can be used for both the index validation and bounding box construction - the validate_indices function was small and only used here, regardless.

Reply:
Seems reasonable.